### PR TITLE
[scroll-animations-1] Update explainer to use new API and concepts.

### PR DESCRIPTION
**Pull request by flackr**
The scroll-animations-1 explainer had gotten fairly out of date after the most recent declarative rethinking and subsequent rewrite of the spec. This brings the concepts and examples in the explainer up to date, though further refinement could be done to focus more on the CSS side of things.

### CODE REVIEW

File: scroll-animations-1/EXPLAINER.md [hunk 1]
```diff
@@ -213,47 +213,37 @@ div.circle {
...
-  ),
-  revealTimeline
+const animation = image.animate(
+  { opacity: [0, 1] },
+  {
+    timeline: revealTimeline,
+    fill: 'both',
+    // The animation starts at the offset that corresponds with image starting
+    // to enter scrollport (i.e., 0% intersect with scroller at its “end” edge).
+    rangeStart: 'enter 0%',
```
Comment (kevers-google) [inline scroll-animations-1/EXPLAINER.md:241]: entry

File: scroll-animations-1/EXPLAINER.md [hunk 2]
```diff
@@ -213,47 +213,37 @@ div.circle {
...
+    // The animation ends at the offset that corresponds with image becoming
+    // fully visible (i.e., 100% intersect with scroller at its “end” edge).
+    rangeEnd: 'enter 100%'
```
Comment (kevers-google) [inline scroll-animations-1/EXPLAINER.md:244]: entry

File: scroll-animations-1/EXPLAINER.md [hunk 3]
```diff
@@ -287,32 +276,28 @@ document.querySelector('.header > .background').animate({
...
-        end: {target: headers[i], edge: 'end', threshold: 1, rootMargin: '-20px'},
-    });
-
     headers[i].animate([
         {transform: 'translateX(-10px)', opacity: 0},
         {transform: 'none', opacity: 1}],
         {
-          duration: 10000,
+          rangeStart: 'enter 0%',
+          rangeEnd: 'enter 100%',
```
Comment (kevers-google) [inline scroll-animations-1/EXPLAINER.md:298]: entry

**Review by kevers-google — CHANGES_REQUESTED** [review]
A few of the examples are still using 'enter" instead of "entry'.

**Comment by flackr** [comment]
> A few of the examples are still using 'enter" instead of "entry'.

Thanks, updated all of the enter references to entry.

**Review by kevers-google — APPROVED** [review]

**Pull request merged**